app/docs/src/reference/management.md

```diff
@@ -0,0 +1,352 @@
+          }[];
+        };
+        parent?: string;
+      };
+    };
+    actions?: {
+      [key: string]: {
+        add?: ("create" | "update" | "refresh" | "delete" | string)[];
+        remove?: ("create" | "update" | "refresh" | "delete" | string)[];
+      };
+    };
+  };
+  message?: string | null;
+};
+```
+
+The `geometry` properties are deprecated and no longer used.
+
+## Common Use Cases
+
+Management functions are commonly used for:
+
+### Importing Existing Resources
+
+Import functions discover resources that already exist in your cloud provider
+and create corresponding components in System Initiative with their current
+configuration.
+
+For example, an AWS VPC import function would:
+
+1. Query AWS for VPC details using a resource ID
+2. Create or update a VPC component with the discovered configuration
+3. Switch the component from "create" to "refresh" actions
+4. Return the imported configuration
+
+### Creating Templates
+
+Template functions generate complete infrastructure patterns by creating
+multiple related components with proper connections and configuration.
+
+For example, a "VPC Template" function might:
+
+1. Create a VPC component
+2. Create public and private subnets as children
+3. Create route tables and make subscriptions to them
+4. Create an internet gateway and make subscriptions to it
+
+### Configuring Multiple Components
+
+Management functions can update many components at once based on policies or
+patterns.
+
+For example, a "Tag Manager" function could:
+
+1. Iterate through all connected components
+2. Apply a standard set of tags to each
+3. Update component properties in bulk
+4. Ensure consistent tagging across infrastructure
+
+## Component Creation
+
+When creating components, you specify:
+
+- `kind`: The schema name (like "VPC" or "Route Table")
+- `properties`: Initial property values
+- `parent`: Optional parent component ID for nested components
+- `connect`: Optional connections to create
+
+Each created component is keyed by an arbitrary ID you choose (like "vpc" or
+"subnet1") that you can reference in connections or parent relationships.
+
+## Component Updates
+
+When updating components, you specify:
+
+- Component ID as the key
+- `properties`: Properties to update (merged with existing)
+- `connect`: Connections to add or remove
+- `parent`: New parent component ID
+
+## Action Management
+
+Management functions can enqueue or remove actions:
+
+```typescript
+actions: {
+  self: {
+    remove: ["create"],
+    add: ["refresh"],
+  },
+  "component-id": {
+    add: ["update"]
+  }
+}
+```
+
+Use "self" to modify actions for the management component itself.
+
+## Management Edges
+
+Components with management functions can have "management edges" to other
+component types. These edges define which types of components the function is
+allowed to manage, providing type safety and clear boundaries for what each
+management function can control.
```

Comment:
Same comment as before about "edges". We can use "relationships" as a rough translation, as needed, but generally speaking a component manages or can manage another one.